bundle/Core/Exception/InvalidStateException.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\IbexaScheduledVisibilityBundle\Core\Exception;
+
+use Exception;
+use Ibexa\Contracts\Core\Repository\Values\Content\Content;
+
+use function sprintf;
+
+final class InvalidStateException extends Exception
+{
+    public function __construct(Content $content, ?Exception $previous = null)
+    {
+        $message = sprintf("Content '%s' with id #%d is not valid for scheduled visibility update.", $content->getName(), $content->getId());
```

Comment:
There is not `getId()` method on the `Content` object, the ID is accessed through the property.

Reply:
![Screenshot from 2024-06-26 11-26-03](https://github.com/netgen/ibexa-scheduled-visibility/assets/56323188/e4449c77-dca9-40e8-938d-de99e6ab59cd)
I don't know if I am watching this correctly...

Reply:
My bad, I was testing it on 4.5. Thinking we should support 4.5 as it could come useful and seems to be already there.

Reply:
Sounds good :+1:

Reply:
fixed in 74e1ff53186b09d0c508639d7188ec21d60c4a96